metagamer/environments/tictactoe.py

```diff
@@ -17,6 +17,28 @@
 logger = logging.getLogger(__name__)
 
 
+def policy_page_lines(board: np.array, player: int = 1) -> tuple:
+    """
+    return a possible position that hasn't yet been played.
+    This policy tries simply to fill the board from the top left.
```

Comment:
suggestion: Might want to add the path it chooses - in this case, it goes left to right, then top to bottom.

Reply:
Good idea